config/prow/plugins.yaml

```diff
@@ -66,6 +66,10 @@ approve:
   - kubernetes/utils
   require_self_approval: false
   lgtm_acts_as_approve: true
+- repos:
+  - kubernetes/website
+  require_self_approval: true
+  lgtm_acts_as_approve: false
```

Comment:
- even if you're an approver, someone needs to approve your PR (it's not automatically approved)
- we don't treat _/lgtm_ as approval if you're an approver; that could be surprising - it has caught me out
- approvers can still explicitly /lgtm and /approve the same PR - eg fine for typo fixes

Reply:
For context: https://kubernetes.slack.com/archives/C1J0BPD2M/p1603222339171000 & following thread